.swiftlint.yml

```diff
@@ -1,6 +1,6 @@
-line_length:
-  warning: 150
-  ignores_comments: true
 
 disabled_rules:
-  identifier_name
+- identifier_name
+- trailing_whitespace
+- line_length
```

Comment:
For what it's worth, there's an option in Xcode Preferences->Text Editing->Editing -- make sure the bottom of the two checkboxes in the following screenshot is checked: https://cl.ly/0d3C2E2B3x0A